design/ceph/object/zone.md

```diff
@@ -80,9 +81,12 @@ After an ceph-object-store is configured to be in this ceph-object-zone, the all
 
 At the moment creating an ceph-object-zone resource only handles Day 1 initial configuration for the zone.
 
-Changes made to the resource's configuration or deletion of the resource are not reflected on the Ceph cluster.
+By default when the ceph-object-zone resource is deleted or modified, the zone or pools is not deleted from the Ceph cluster.
 
-To be clear, when the ceph-object-zone resource is deleted or modified, the zone is not deleted from the Ceph cluster. Zone deletion must be done through the toolbox.
+If user disables `preservePoolsOnDelete` then zone or pool will be removed only if following conditions satisfied:
+- all the objects belong to this zone need to deleted first.
+- it should not be master zone.
+- if it is master zone, then no other zones belongs to zonegroup.
```

Comment:
Does this mean deletion behavior of the zone itself is different depending on the `preservePoolsOnDelete` setting? That doesn't seem quite right. That setting should just affect pools (and be mentioned in the paragraph above). The behavior of deleting the zone itself can be talked about here.